scripts/assert-codegen-build-committed-actions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Stackoverflow: How do I programmatically determine if there are uncommitted changes?
+# https://stackoverflow.com/questions/3878624/how-do-i-programmatically-determine-if-there-are-uncommitted-changes
+
+IS_CI="${CI:-false}"
+
+if [ "$IS_CI" = true ]; then
+  git config --local url."https://github.com/".insteadOf git@github.com:
+  git config --local user.name "Medusajs Bot"
+  git config --local user.email "core@medusa-commerce.com"
+fi
+
+FILE_CHANGES=$(git status --porcelain=v1 | grep 'packages/generated')
+FILE_CHANGES_COUNT=$(git status --porcelain=v1 | grep 'packages/generated' | wc -l)
```

Comment:
The strategy to verify is codegen build has been commited
https://git-scm.com/docs/git-status#Documentation/git-status.txt---porcelainltversiongt
https://www.baeldung.com/linux/bash-count-lines-in-file#wc